README.md

```diff
@@ -1,12 +1,89 @@
+  - Click 'Create a resource', then search the marketplace for 'SQL Database' and select it. Provide a 'Subscription', 'Resource Group', and 'Database name.' Under the 'Server' field, click 'Create New'
+<kbd>![alt text](/Images/dbSetup.png)</kbd>
+  - Fill in the fields of the 'New server' panel. Make sure you know your 'Server admin login' and 'Password' as you will need them later. Click 'OK' at the bottom of the panel.
+  - Click 'Review and Create' at the bottom of the page. Then press 'Create.' While you are waiting for your resource to be created, feel free to do the "Set Up Your Local Environment" step and return here when completed.
+- Once created, navigate to the SQL Database resource. In the left panel, click 'Query editor'
+- Enter your login from when you created the SQL Database.
+  - If an error pops up for not being able to open the server, copy the Client IP address in the second sentence of the error message, and click 'set server firewall' at the bottom
+  - In the new window, click 'Add Client IP.' This will create an entry
+  - In the section with Rule Name, Start IP, and End IP, paste the IP address you just copied into the Start and End IP fields for the entry created in the previous step.
+  - Hit 'Save' in the top left and navigate back into the SQL Database login page.
+  - Enter your login. You should now be in the Query editor view
+- If you plan on skipping the tutorial, go to 'Set Primary Key.' Continue if you are doing the tutorial.
+- Enter the below script and hit run to create a table. Once created, if you expand the Tables section by clicking the arrow, you should see a table
+
+    ```sql
+    CREATE TABLE Employees (
+          EmployeeID int,
+          FirstName varchar(255),
+          LastName varchar(255),
+          Company varchar(255),
+          Team varchar(255)
+    );
+    ```
+
+- Enter the blow script and hit run to create an entry in the table. Once created, if you right click your table name and click 'Select Top 1000 Rows', you'll be able to see your entry present.
+
+    ```sql
+    INSERT INTO [dbo].[Employees] values (1, 'Hello', 'World', 'Microsoft', 'Functions')
+    ```
+
+- Congratulations! You have successfully setup an Azure SQL Database!
+
+### Set Primary Key ###
+
+A primary key must be set in your SQL table before using the bindings. To do this run the below SQL commands in the SQL query editor.
+
+1. Ensure no NULL values in primary key column
+
+    ```sql
+    ALTER TABLE [dbo].[Employees] alter column EmployeeId int NOT NULL
+    ```
+
+1. Set primary key column
 
-This repository contains extension code for SQL Server input and output bindings. Trigger bindings are coming soon.
+    ```sql
+    ALTER TABLE [dbo].[Employees] ADD CONSTRAINT PKey PRIMARY KEY CLUSTERED (EmployeeId);
+    ```
 
-# Input and Output Binding
-## Quick Start
+### Set Up Local .NET Function App ###
+
+**NOTE: THE MYGET PACKAGE DOES NOT CURRENTLY CONTAIN TRIGGER BINDING FUNCTIONALITY**
```

Comment:
This is no longer accurate, right?

Reply:
I had to modify .csproj in order to get the SQL trigger working. I tried it out again this morning and just using the myget package resulted in some namespaces not being found for the trigger. I've attached a screenshot below. Is the myget supposed to include full functionality for the trigger now? I might have missed something or taken a weird step in setup, so if someone can verify, that would be great. 

<img width="889" alt="Capture" src="https://user-images.githubusercontent.com/69606408/94586032-9c13d600-0235-11eb-9cb2-1c269c55cd86.PNG">

Reply:
Fixed with preview2